File: src/bin/day0.rs
const INPUT: &str = include_str!("inputs/day0.txt");

pub fn day0(input: &str) -> (usize, usize) {
    (input.len(), input.len())
}

fn main() {
    let (p1, p2) = day0(INPUT);
    println!("{}\n{}", p1, p2);
}

#[cfg(test)]
mod tests {
    use super::*;

    #[test]
    fn test1() {
        let input = include_str!("inputs/day0.test1.txt");
        let (p1, p2) = day0(input);

        assert_eq!(p1, input.len());
        assert_eq!(p2, input.len());
    }

    #[test]
    fn test_main() {
        let (p1, p2) = day0(INPUT);

        assert_eq!(p1, INPUT.len());
        assert_eq!(p2, INPUT.len());
    }
}

File: python/day0.py
from pprint import pprint

def day0(input):
    acc1 = len(input)
    acc2 = len(input)
    return acc1,acc2

input = open("src/bin/inputs/day0.test1.txt", "r").read()
pprint(day0(input.strip()))


input = open("src/bin/inputs/day0.txt", "r").read()
pprint(day0(input.strip()))
